### PR TITLE
Infect Suicide CT

### PR DESCRIPTION
**Pull request by Darnias**

### CODE REVIEW

**Review by tilgep — APPROVED** [review]

File: scripts/vscripts/ZombieReborn.lua [hunk 1]
```diff
@@ -73,6 +75,12 @@ function OnPlayerDeath(event)
     if hAttacker:GetTeam() == CS_TEAM_T and hVictim:GetTeam() == CS_TEAM_CT then
         DoEntFireByInstanceHandle(hVictim, "runscriptcode", "Infect(nil, thisEntity, true)", 0, nil, nil)
     end
+
+    -- Infect Humans that died after first infection has started
+    if ZR_ROUND_STARTED and ZR_ZOMBIE_SPAWNED and hVictim:GetTeam() == CS_TEAM_CT then
+        --print("infecting suicide")
+        Infect(nil, hVictim, bSpawnType)
```
Comment (EasterLee) [inline scripts/vscripts/ZombieReborn.lua:82]: Wouldn't this cause duplicate killfeed?

Comment (Darnias) [reply scripts/vscripts/ZombieReborn.lua:82]: It does, but SetTeam also duplicates the killfeed.

Comment (EasterLee) [reply scripts/vscripts/ZombieReborn.lua:82]: Not if you setteam while they are actually dead. Try delaying it by one tick using EntFire.

Comment (Darnias) [reply scripts/vscripts/ZombieReborn.lua]: Fixed

**Review by EasterLee — APPROVED** [review]

**Review by notkoen — APPROVED** [review]

**Pull request merged**